units/src/weight.rs

```diff
@@ -55,8 +55,15 @@ impl Weight {
     pub fn from_kwu(wu: u64) -> Option<Self> { wu.checked_mul(1000).map(Weight) }
 
     /// Constructs a new [`Weight`] from virtual bytes, returning [`None`] if an overflow occurred.
-    pub fn from_vb(vb: u64) -> Option<Self> {
-        vb.checked_mul(Self::WITNESS_SCALE_FACTOR).map(Weight::from_wu)
+    pub const fn from_vb(vb: u64) -> Option<Self> {
+        let weight_unit = vb.checked_mul(Self::WITNESS_SCALE_FACTOR);
+        // replace with ? when MSRV supports it in const context
+        // https://github.com/rust-lang/rust/issues/74935
```

Comment:
Do we do `// TODO` or `// FIXME`?
If yes I prefer those since it is easy to grep it.

Reply:
TODO is good I think.  Updated.